src/admin-views/editor/panel/ticket.php

```diff
@@ -189,7 +216,132 @@ class="ticket_field ticket_provider"
 				 */
 				do_action( 'tribe_events_tickets_metabox_edit_main', $post_id, $ticket_id ); ?>
 			</section>
-
+			<div class="input_block">
+				<label class="ticket_form_label ticket_form_left" for="ticket_description"><?php esc_html_e( 'Description:', 'event-tickets' ); ?></label>
+				<textarea
+						rows="5"
+						cols="40"
+						name="ticket_description"
+						class="ticket_field ticket_form_right"
+						id="ticket_description"
+				><?php echo esc_textarea( $ticket ? $ticket->description : null ) ?></textarea>
+				<div class="input_block">
+					<label class="tribe_soft_note">
+						<input
+								type="checkbox"
+								id="tribe_tickets_show_description"
+								name="ticket_show_description"
+								value="1"
+								class="ticket_field ticket_form_left"
+								<?php checked( true, $ticket ? $ticket->show_description : true ); ?>
+						>
+						<?php
+						echo esc_html( sprintf(
```

Comment:
:no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line

<sub>hash: 8cefbe77b5fce0d09070d1e017d49a8d</sub>